src/modules/flow/iio/iio.json

```diff
@@ -24,6 +24,15 @@
         "close": "gyroscope_close",
         "open": "gyroscope_open"
       },
+      "node_type": {
+        "access": [
+          "base"
+        ],
+        "data_type": "struct iio_node_type",
+        "extra_methods": {
+          "out_port": "SOL_FLOW_NODE_TYPE_IIO_GYROSCOPE__OUT__OUT"
```

Comment:
Here you could define some `type` or `iio_type` and set to `IIO_GYROSCOPE`. Then you could have a `common_open` for all node types.